scripts/run_locally_with_docker.sh

```diff
@@ -21,10 +20,9 @@ else
   EXPOSED_PORTS=""
 fi
 
-docker run -it --rm \
+docker run -it -d --name notifications_api --rm \
   -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
   -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
-  -e SQLALCHEMY_DATABASE_URI=$SQLALCHEMY_DATABASE_URI \
```

Comment:
Are we sure we want to remove this? Worth double checking

Reply:
ok, since we are not using the make command i created, i reverted the script to its original format